package.json

```diff
@@ -42,50 +42,49 @@
     "@native-html/iframe-plugin": "^2.6.1",
     "@native-html/table-plugin": "^5.3.1",
     "@react-native-async-storage/async-storage": "1.23.1",
-    "@react-native-community/datetimepicker": "8.0.1",
-    "@react-native-community/netinfo": "11.3.1",
-    "@react-native-masked-view/masked-view": "0.3.1",
+    "@react-native-community/datetimepicker": "8.2.0",
+    "@react-native-community/netinfo": "11.4.1",
+    "@react-native-masked-view/masked-view": "0.3.2",
     "@react-navigation/bottom-tabs": "^6.5.9",
     "@react-navigation/drawer": "^6.6.4",
     "@react-navigation/elements": "^1.3.19",
     "@react-navigation/native": "^6.1.8",
     "@react-navigation/stack": "^6.3.18",
     "@reactvision/react-viro": "^2.41.6",
-    "@sentry/react-native": "~5.24.3",
-    "@shopify/flash-list": "1.6.4",
+    "@sentry/react-native": "~6.3.0",
+    "@shopify/flash-list": "1.7.1",
     "apollo-cache-inmemory": "1.6.6",
     "apollo-cache-persist": "~0.1.1",
     "apollo-client": "2.6.10",
     "apollo-link": "~1.2.12",
     "apollo-link-context": "1.0.20",
     "apollo-link-http": "1.5.17",
     "apollo-link-rest": "~0.7.3",
-    "expo": "~51.0.39",
-    "expo-application": "~5.9.1",
-    "expo-asset": "~10.0.10",
-    "expo-av": "~14.0.7",
-    "expo-barcode-scanner": "~13.0.1",
-    "expo-blur": "~13.0.2",
-    "expo-build-properties": "~0.12.5",
-    "expo-camera": "~15.0.16",
-    "expo-clipboard": "~6.0.3",
-    "expo-constants": "~16.0.2",
-    "expo-dev-client": "~4.0.29",
-    "expo-device": "~6.0.2",
-    "expo-document-picker": "~12.0.2",
-    "expo-font": "~12.0.10",
-    "expo-image-picker": "~15.0.7",
-    "expo-linear-gradient": "~13.0.2",
-    "expo-linking": "~6.3.1",
-    "expo-localization": "~15.0.3",
-    "expo-location": "~17.0.1",
-    "expo-media-library": "~16.0.5",
-    "expo-notifications": "~0.28.19",
-    "expo-screen-orientation": "~7.0.5",
-    "expo-secure-store": "~13.0.2",
-    "expo-splash-screen": "~0.27.7",
-    "expo-status-bar": "~1.12.1",
-    "expo-updates": "~0.25.27",
+    "expo": "^52.0.0",
+    "expo-application": "~6.0.2",
+    "expo-asset": "~11.0.2",
+    "expo-blur": "~14.0.2",
+    "expo-build-properties": "~0.13.2",
+    "expo-camera": "~16.0.11",
+    "expo-clipboard": "~7.0.1",
+    "expo-constants": "~17.0.4",
+    "expo-dev-client": "~5.0.9",
+    "expo-device": "~7.0.2",
+    "expo-document-picker": "~13.0.2",
+    "expo-font": "~13.0.3",
+    "expo-image-picker": "~16.0.4",
+    "expo-linear-gradient": "~14.0.2",
+    "expo-linking": "~7.0.4",
+    "expo-localization": "~16.0.1",
+    "expo-location": "~18.0.5",
+    "expo-media-library": "~17.0.5",
+    "expo-notifications": "~0.29.12",
+    "expo-screen-orientation": "~8.0.4",
+    "expo-secure-store": "~14.0.1",
+    "expo-splash-screen": "~0.29.20",
```

Comment:
> Splash screens for Android cannot be fullscreen with this API (and this did not work particularly well before either), so you may need to update your splash screen. 

can this be something we need to check deeper? https://expo.dev/changelog/2024/11-12-sdk-52#big-improvements-to-expo-splash-screen

Reply:
> The splash key will be deprecated in favor of the config plugin in a future release. For now, expo-splash-screen will attempt to map your splash config to the equivalent configuration for the config plugin automatically. For now, you won't need to make any changes to your existing splash configuration on iOS. However, on Android you will need to migrate away from using a full screen splash image if you are currently using one, and instead use a smaller image such as your app icon.

Reply:
Yes. That's going to be a problem for us. I've been trying to figure out how to solve it. If you tried the new development build you can see it. SplashScreen is too small. It looks like a logo. Maybe we can solve this problem by adding a standard width. But it needs to be tested.

Reply:
ok, so this will be the last thing to figure put, right?

Reply:
I do not understand to develop it in App.js. iOS legacy is not an option as it will be removed soon in the future.
Please make research how to do splash screens in the latest version and  make some proposals.

Reply:
It was updated in this way because we decided to add the app icon and primary color code as a background instead of a full screen splash screen.

[465e56d](https://github.com/smart-village-solutions/smart-village-app-app/pull/1230/commits/465e56d7db4d5a822c3151b10a85cc0632c249a1)

Reply:
|ios|android|
|--|--|
![Simulator Screenshot - iPhone 16 Pro Max - 2025-01-17 at 14 41 31](https://github.com/user-attachments/assets/2954d1de-8007-40df-b1a2-26b38e0e487c)|![Screenshot_20250117-144602](https://github.com/user-attachments/assets/5ec6ca50-5e27-4c0f-9c34-0935b4848225)

Reply:
please use white for our dev app as background color. primary makes no sense here.

and for android the size of the logo needs to be corrected, so that it does not get cut.

Reply:
Yes, it is. That's better.

![Screenshot_20250117-150926](https://github.com/user-attachments/assets/a5865b48-6c5a-456e-a85a-2d81f7868837)